core/controllers/learner_dashboard.py

```diff
@@ -175,7 +175,9 @@ def get(self, thread_id):
                 'description': suggestion_thread.subject,
                 'author_username': authors_settings[0].username,
                 'author_picture_data_url': (
-                    authors_settings[0].profile_picture_data_url)
+                    authors_settings[0].profile_picture_data_url),
+                'created_on': utils.get_time_in_millisecs(
```

Comment:
Can we add a simple test for this value (maybe the handler returns this key in the dict)? Also, do you know the reason behind this issue? is it because of any regression? or it wasn't implemented correctly initially?

Reply:
I think it wasn't implemented corrected initially.